docs/pages/2020_Intro_Week/sections/running.rst

```diff
@@ -4,3 +4,614 @@
+
+.. figure:: include/images/verdi_graph/si/graph-full.png
+   :alt:
+
+In order to check which input files AiiDA creates, we can perform a *dry run* of the submission process.
+Let's tell the builder that we want a dry run and that we don't want to store the provenance of the dry run:
+
+.. code:: python
+
+    builder.metadata.dry_run = True
+    builder.metadata.store_provenance = False
+
+It's time to run:
+
+.. code:: python
+
+    from aiida.engine import run
+    run(builder)
+
+.. note::
+
+   Instead of using the builder, you can also simply pass the calculation class as the first argument, followed by the inputs as keyword arguments, e.g.:
+
+   .. code:: python
+
+      run(PwCalculation, structure=structure, pseudos={'Si': pseudo_node}, ....)
+
+   The builder is simply a convenience wrapper providing tab-completion in the shell and automatic help strings.
+
+This creates a folder of the form ``submit_test/[date]-0000[x]`` in the current directory.
+In your second terminal:
+
+ * open the input file ``aiida.in`` within this folder
+ * compare it to input data nodes you created earlier
+ * verify that the `pseudo` folder contains the needed pseudopotentials
+ * have a look at the submission script ``_aiidasubmit.sh``
+
+.. note::
+
+   The files created by a dry run are only intended for  inspection
+   and cannot be used to correct the inputs of your calculation.
+
+Submitting the calculation
+--------------------------
+
+Up to now we've just been playing around and our calculation has been kept in memory and not in the database.
+Now that we have inspected the input files and convinced ourselves that Quantum ESPRESSO will have all the information it needs to perform the calculation, we will submit the calculation properly.
+Doing so will make sure that all inputs are stored in the database, will run and store the calculation and link the outputs to it.
+
+Let's revert the following values in our builder to their defaults:
+
+.. code:: python
+
+    builder.metadata.dry_run = False
+    builder.metadata.store_provenance = True
+
+And then rely on the submit machinery of AiiDA,
+
+.. code:: python
+
+    from aiida.engine import submit
+    calculation = submit(builder)
+
+As soon as you have executed these lines, the ``calculation`` variable contains a ``PwCalculation`` instance, already submitted to the daemon.
+
+.. note::
+
+   You may have noticed that we used ``submit`` here instead of ``run``.
+   The difference is that ``submit`` will hands over the calculation to the daemon running in the background, while ``run`` will execute all tasks in the current shell.
+
+   All processes in AiiDA (you will soon get to know more) can be "launched" using one of available functions:
+
+    * run
+    * run_get_node
+    * run_get_pk
+    * submit
+
+   which are explained in more detail in the `online documentation <https://aiida-core.readthedocs.io/en/latest/working/processes.html#launching-processes>`_.
```

Comment:
Ok, I linked to the specific part of the topics that describes those 4 methods ([here](https://aiida-quantumespresso.readthedocs.io/en/stable/user_guide/calculation_plugins/pw.html)).

Reply:
Wait, now that I see the issue you linked more clearly I'm not sure I used the right link. Where do I include the tag v1.3.0 in the following url? Instead of `/latest/` doesn't work...

https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/processes/usage.html?highlight=run_get_pk#launching-processes

Reply:
well it won't work yet, because v1.3.0 is not released yet, but this works: https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/

Reply:
Ok, what do you prefer then? Leave the link as it is pointing to the latest doc? or leave a currently broken link to v1.3.0?

Reply:
yep I think the broken link is best; so we don't forget